Please is PSTCKRSA missing in the static framework release?

Hello Ibrahim,

I didn't want to leave this on the official home for the repo because it's more of a clarification than an issue. I was trying to add iOS support to cordova-plugin-paystack using the static `Paystack.framework` but I keep running into the errors each time I attempt to build the app:

`Undefined symbols for architecture i386:
  "_OBJC_CLASS_$_PSTCKRSA", referenced from:
      objc-class-ref in Paystack(PSTCKCardParams.o)
ld: symbol(s) not found for architecture i386
`

Please could it be possible that the interface and implementation for `PSTCKRSA` were not bundled with the module in the static framework release at all? Thanks

--- a/Paystack.xcodeproj/project.pbxproj
+++ b/Paystack.xcodeproj/project.pbxproj
@@ -239,14 +239,15 @@
 		04F213371BCECB1C001D6F22 /* PSTCKAPIResponseDecodable.h in Headers */ = {isa = PBXBuildFile; fileRef = 04F213341BCECB1C001D6F22 /* PSTCKAPIResponseDecodable.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		04FCFA191BD59A8C00297732 /* PSTCKCategoryLoader.h in Headers */ = {isa = PBXBuildFile; fileRef = 04FCFA171BD59A8C00297732 /* PSTCKCategoryLoader.h */; };
 		04FCFA1A1BD59A8C00297732 /* PSTCKCategoryLoader.m in Sources */ = {isa = PBXBuildFile; fileRef = 04FCFA181BD59A8C00297732 /* PSTCKCategoryLoader.m */; };
+		101987811CA47DAC0089A68A /* PSTCKRSA.m in Sources */ = {isa = PBXBuildFile; fileRef = 10500AB11C8135D800EEF7CF /* PSTCKRSA.m */; };
 		10500AB21C8135D800EEF7CF /* PSTCKRSA.m in Sources */ = {isa = PBXBuildFile; fileRef = 10500AB11C8135D800EEF7CF /* PSTCKRSA.m */; };
 		10500AB51C8136EF00EEF7CF /* PSTCKRSATest.m in Sources */ = {isa = PBXBuildFile; fileRef = 10500AB41C8136EF00EEF7CF /* PSTCKRSATest.m */; };
-		10500ABB1C82155B00EEF7CF /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; };
+		10500ABB1C82155B00EEF7CF /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		10500ABC1C82157A00EEF7CF /* PSTCKRSA.m in Sources */ = {isa = PBXBuildFile; fileRef = 10500AB11C8135D800EEF7CF /* PSTCKRSA.m */; };
-		10500ABD1C8215F400EEF7CF /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; };
+		10500ABD1C8215F400EEF7CF /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		10500ABE1C8216C200EEF7CF /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; };
 		10A653A31C88CC5900EBC974 /* PSTCKCardParams.h in Headers */ = {isa = PBXBuildFile; fileRef = 04CDE5BB1BC1F21500548833 /* PSTCKCardParams.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		10FC52071C88DDB3004A0733 /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; };
+		10FC52071C88DDB3004A0733 /* PSTCKRSA.h in Headers */ = {isa = PBXBuildFile; fileRef = 10500AB31C8135F800EEF7CF /* PSTCKRSA.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		C1718D561C3B2E5B002A7CB3 /* UIImage+Paystack.h in Headers */ = {isa = PBXBuildFile; fileRef = C1718D541C3B2E5B002A7CB3 /* UIImage+Paystack.h */; };
 		C1718D571C3B2E5B002A7CB3 /* UIImage+Paystack.m in Sources */ = {isa = PBXBuildFile; fileRef = C1718D551C3B2E5B002A7CB3 /* UIImage+Paystack.m */; };
 		C1718D581C3B2E60002A7CB3 /* UIImage+Paystack.h in Headers */ = {isa = PBXBuildFile; fileRef = C1718D541C3B2E5B002A7CB3 /* UIImage+Paystack.h */; };
@@ -706,9 +707,9 @@
 				049E84EB1A605EF0000B66CD /* PSTCKToken.h in Headers */,
 				04E32AA01B7A9490009C9E35 /* PSTCKPaymentCardTextField.h in Headers */,
 				0438EF491B74183100D506CC /* PSTCKCardBrand.h in Headers */,
+				10500ABD1C8215F400EEF7CF /* PSTCKRSA.h in Headers */,
 				049952D61BCF14930088C703 /* PSTCKAPIPostRequest.h in Headers */,
 				04F213331BCEAB61001D6F22 /* PSTCKFormEncodable.h in Headers */,
-				10500ABD1C8215F400EEF7CF /* PSTCKRSA.h in Headers */,
 				049E84EC1A605EF0000B66CD /* PaystackError.h in Headers */,
 				C1718D581C3B2E60002A7CB3 /* UIImage+Paystack.h in Headers */,
 				04FCFA191BD59A8C00297732 /* PSTCKCategoryLoader.h in Headers */,
@@ -735,8 +736,8 @@
 				049952CF1BCF13510088C703 /* PSTCKAPIPostRequest.h in Headers */,
 				0438EF471B74183100D506CC /* PSTCKCardBrand.h in Headers */,
 				10A653A31C88CC5900EBC974 /* PSTCKCardParams.h in Headers */,
-				04CDB5161A5F30A700B854EE /* PaystackError.h in Headers */,
 				10FC52071C88DDB3004A0733 /* PSTCKRSA.h in Headers */,
+				04CDB5161A5F30A700B854EE /* PaystackError.h in Headers */,
 				04E32A9D1B7A9490009C9E35 /* PSTCKPaymentCardTextField.h in Headers */,
 				04F213311BCEAB61001D6F22 /* PSTCKFormEncodable.h in Headers */,
 				04CDB5021A5F30A700B854EE /* PSTCKFormEncoder.h in Headers */,
@@ -762,8 +763,8 @@
 				04D12C3E1A5F55D10010446E /* PSTCKToken.h in Headers */,
 				04D12C3F1A5F55D10010446E /* PaystackError.h in Headers */,
 				049952D51BCF14920088C703 /* PSTCKAPIPostRequest.h in Headers */,
-				0438EFDB1B7524AA00D506CC /* PSTCKCardBrand.h in Headers */,
 				10500ABB1C82155B00EEF7CF /* PSTCKRSA.h in Headers */,
+				0438EFDB1B7524AA00D506CC /* PSTCKCardBrand.h in Headers */,
 				0433EB4A1BD06313003912B4 /* NSDictionary+Paystack.h in Headers */,
 				04F213361BCECB1C001D6F22 /* PSTCKAPIResponseDecodable.h in Headers */,
 				04D12C3A1A5F55D10010446E /* PSTCKFormEncoder.h in Headers */,
@@ -1098,6 +1099,7 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				101987811CA47DAC0089A68A /* PSTCKRSA.m in Sources */,
 				0438EF451B74170D00D506CC /* PSTCKCardValidator.m in Sources */,
 				0438EF311B7416BB00D506CC /* PSTCKFormTextField.m in Sources */,
 				04FCFA1A1BD59A8C00297732 /* PSTCKCategoryLoader.m in Sources */,

--- a/Paystack.xcodeproj/xcshareddata/xcschemes/PaystackOSX.xcscheme
+++ b/Paystack.xcodeproj/xcshareddata/xcschemes/PaystackOSX.xcscheme
@@ -66,7 +66,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       launchStyle = "0"

--- a/Paystack.xcodeproj/xcshareddata/xcschemes/PaystackiOSStaticFramework.xcscheme
+++ b/Paystack.xcodeproj/xcshareddata/xcschemes/PaystackiOSStaticFramework.xcscheme
@@ -33,7 +33,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       launchStyle = "0"

--- a/Paystack/PublicHeaders/PSTCKAPIClient.h
+++ b/Paystack/PublicHeaders/PSTCKAPIClient.h
@@ -5,7 +5,7 @@
 
 #import <Foundation/Foundation.h>
 
-static NSString *const __nonnull PSTCKSDKVersion = @"0.1.0";
+static NSString *const __nonnull PSTCKSDKVersion = @"1.0.0";
 
 @class PSTCKCard, PSTCKCardParams, PSTCKToken;
 

--- a/Paystack/PublicHeaders/Paystack.h
+++ b/Paystack/PublicHeaders/Paystack.h
@@ -17,6 +17,7 @@
 #import "PSTCKCardValidationState.h"
 #import "PSTCKCardValidator.h"
 #import "PSTCKToken.h"
+#import "PSTCKRSA.h"
 
 #if TARGET_OS_IPHONE
 #import "PSTCKPaymentCardTextField.h"

